recipes-devtools/triton/triton-python-backend_r22.05.bb

```diff
@@ -0,0 +1,48 @@
+DESCRIPTION = "Triton backend that enables pre-process, post-processing and other logic to be implemented in Python."
+LICENSE = "BSD-3-Clause"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=7889848dc86811b06ded7bfa9ba39e02"
+
+SRC_URI = "git://github.com/triton-inference-server/python_backend.git;protocol=https;branch=r22.05 \
+           file://0001-fix-cmake-build.patch \
+           "
+
+SRCREV = "6133572a4f090721cc52e79595b70f383397b186"
+
+S = "${WORKDIR}/git"
+
+DEPENDS = " \
+    zlib \
+    libarchive \
+    boost \
+    dlpack \
+    triton-common \
+    triton-core \
+    triton-backend \
+"
+
+RDEPENDS_${PN} = " \
+    python3-pybind11 \
+    python3-gevent \
+    python3-geventhttpclient \
+    python3-rapidjson \
+    dlpack \
```

Comment:
You should remove `dlpack` from `RDEPENDS` list ?

Reply:
@ichergui You are right